profiles/common/modules/features/cce_basic_config/cce_basic_config.install

```diff
@@ -1910,3 +1910,25 @@ function cce_basic_config_update_7234() {
     }
   }
 }
+
+/**
+ * Restore user_administrator.
+ */
+function cce_basic_config_update_7235() {
+  $account = user_load_by_name('user_administrator');
+
+  if (empty($account)) {
+    $account = array(
+      'name' => 'user_administrator',
+      'pass' => 'pass',
+      'mail' => 'administrator@nomail.nodomain.notld',
+      'status' => 1,
+      'init' => 'administrator@nomail.nodomain.notld',
+      'roles' => array(
+        DRUPAL_AUTHENTICATED_RID => 'authenticated user',
```

Comment:
user_administrator should have the administrator role.

Reply:
fixed